modules/users/client/config/referrals.client.routes.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+// Setting up route
+angular.module('referrals').config(['$stateProvider',
+  function ($stateProvider) {
+    // Referrals state routing
+    $stateProvider
+      .state('referrals', {
+        abstract: true,
+        url: '/referrals',
+        template: '<ui-view>',
+        data:{
+          roles:['admin', 'user']
+        }
+      })
+
+      //safetyNet
+      .state('referrals.safetyNet', {
+        url: '/',
+        templateUrl: 'modules/users/client/views/employees/list-referrals.client.view.html',
+      })
+
+      //Both Users
+      .state('referrals.list', {
+        url: '',
+        templateUrl: 'modules/users/client/views/employees/list-referrals.client.view.html',
+        data: {
+          roles: ['admin', 'user']
+        }
+      })
+      //AdminViews
+      .state('referrals.create', {
+        url: '/create',
+        templateUrl: 'modules/users/client/views/admin/create-referral.client.view.html',
+        data: {
+          roles: ['admin']
+        }
```

Comment:
There isn't currently a way to get to this state except by the URL - we should probably have a "create" button somewhere, I'd suggest on the referral list